packages/page-referenda/src/Referenda/Delegate/index.tsx

```diff
@@ -103,8 +103,8 @@ function Delegate ({ className, palletReferenda, palletVote, tracks }: Props): R
     [api, balance, conviction, includeTracks, isAllTracks, palletVote, toAccount, trackId]
   );
 
-  const isStep1Valid = !!(accountId && activityFrom && includeTracks && (includeTracks.length > 0));
-  const isStep2Valid = !!(toAccount && activityTo);
+  const isStep1Valid = !!(accountId && includeTracks && (includeTracks.length > 0));
+  const isStep2Valid = !!(toAccount && activityTo && (activityTo.length > 0));
```

Comment:
In these we actually just want to know that we have received _something_ from the hook, not that the actual values are valid.

So the intent was -

- if `null` the hook has not fired, so wait a bit
- if `[...]` (any values), we have a result

So the original actually looks fine.

Reply:
Understood thanks. I reverted these changes.